src/applications/gi/components/profile/SchoolClosingDetails.jsx

```diff
@@ -23,17 +22,7 @@ const SchoolClosingDetails = ({
           <div>
             <p>{content}</p>
             {schoolWebsite && (
-              <a
-                href={schoolWebsite}
-                target="_blank"
-                rel="noopener noreferrer"
-                onClick={() => {
-                  recordEvent({
-                    event: 'nav-warning-alert-box-content-link-click',
-                    alertBoxHeading: headline,
-                  });
-                }}
-              >
```

Comment:
where is `event` or `alertBoxHeading` being set in https://github.com/department-of-veterans-affairs/component-library/blob/master/src/helpers/analytics.js or https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/AlertBox/AlertBox.jsx ?

Reply:
@zurbergram - the component triggers a CustomEvent on the document.body....

https://github.com/department-of-veterans-affairs/component-library/blob/eaffb602607f3d1b021d0eb2f9101e7c05dc3b28/src/components/AlertBox/AlertBox.jsx#L62

...which is then interpreted into an actual dataLayer event (with the headline) here...

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/component-library-analytics-setup.js